jobs/service-fabrik-broker/templates/config/eventlog-config-internal.yml.erb

```diff
@@ -217,3 +217,11 @@ defaults:
     description: update service fabrik mongodb deployment
     tags:
     - servicefabrikdbupdate
+# API used as a place holder to generate event logs for maas
+"/usage/v2/usage/documents":
```

Comment:
Why is this added here? It's not an API from us right? Rather its an API we call

Reply:
Yeah !! But we need to generate event logs , for that we need a reference api.

https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/561/files#diff-bc78e2131648a34d5d3dbd23d84a02f1R123

Reply:
yesh! @jagadish-kb explained me! thanks.